src/Nethermind/Nethermind.Api/Extensions/IPlugin.cs

```diff
@@ -4,22 +4,23 @@
 using System;
 using System.Threading.Tasks;
 
-namespace Nethermind.Api.Extensions
+namespace Nethermind.Api.Extensions;
+
+public interface INethermindPlugin : IAsyncDisposable
 {
-    public interface INethermindPlugin : IAsyncDisposable
-    {
-        string Name { get; }
+    string Name { get; }
+
+    string Description { get; }
 
-        string Description { get; }
+    string Author { get; }
 
-        string Author { get; }
+    void InitRlpDecoders(INethermindApi api) { }
 
-        Task Init(INethermindApi nethermindApi);
+    Task Init(INethermindApi nethermindApi) => Task.CompletedTask;
 
-        Task InitNetworkProtocol();
+    Task InitNetworkProtocol() => Task.CompletedTask;
 
-        Task InitRpcModules();
+    Task InitRpcModules() => Task.CompletedTask;
 
-        bool MustInitialize { get => false; }
-    }
+    bool MustInitialize => true;
```

Comment:
Changing default value here might be a breaking change? This was `false` so by default an issue in a plugin won't prevent the node from starting.